pwndbg/heap/__init__.py

```diff
@@ -5,19 +5,26 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import pwndbg.heap.dlmalloc
 import pwndbg.heap.heap
-import pwndbg.heap.ptmalloc
+import pwndbg.symbol
 
-current = pwndbg.heap.heap.Heap()
+current_heap = None
 
 @pwndbg.events.new_objfile
 def update():
-    global current
+    import pwndbg.heap.dlmalloc
+    import pwndbg.heap.ptmalloc
 
+    global current_heap
 
-    if pwndbg.symbol.get('ptmalloc_init'):
-      current = pwndbg.heap.ptmalloc.Heap()
 
-    elif pwndbg.symbol.get('malloc_stats'):
-      current = pwndbg.heap.dlmalloc.Heap()
+    if pwndbg.symbol.address('ptmalloc_init'):
+        current_heap = pwndbg.heap.ptmalloc.Heap()
+
+    else:
+        # Default to ptmalloc heap for now until
+        # there are more implementations
+        current_heap = pwndbg.heap.ptmalloc.Heap()
+
+def get_heap():
```

Comment:
I prefer the `current` global which is updated via the event function, to a `getter` function which returns the global.